sympy/solvers/ode/tests/test_single.py

```diff
@@ -1853,6 +1856,20 @@ def _get_examples_ode_sol_2nd_nonlinear_autonomous_conserved():
     }
 
 
+@_add_example_keys
+def _get_examples_ode_sol_lienard():
+    return {
+            'hint': "lienard",
+            'func': f(x),
+            'examples': {
+    'lienard_01': {
+        'eq': f(x).diff(x, x) + 2/x*f(x).diff(x) + f(x),
+        'sol': [Eq(f(x), C1/(x*sqrt(tan(C2 + x)**2 + 1)))],
```

Comment:
Can you come up with any other test cases that work?

Reply:
There are various tests cases that match in paper with the method implemented from wolfram's answers, but the main issue is that the solution of the Riccati equation often give a series solution like for `eq = f(x).diff(x, x)+exp(x)*f(x).diff(x)+f(x) `, `eq = f(x).diff(x, x)+x*f(x).diff(x)+f(x) `. 
[Riccati](https://github.com/sympy/sympy/pull/20936/files/ab0634dbc1d0c7e8ac707da7da0d7a14c8c99264#diff-322899cfb811d22aadc64f3ff65f04b014f8b2c99ba8092036010a758240c346R916), these are solved mainly by the `Lie group method`.

Like for `eq = f(x).diff(x, x)+x*f(x).diff(x)+f(x) `, the riccati equation `eq = f(x).diff(x)+f(x)**2+x*f(x)+1`, it returns a series solution

`Eq(f(x), x*(-C1**2 - 1) + x**3*(3*C1**2 + (-C1**2 - 1)*(6*C1**2 + 1) + 1)/6 + x**5*(-8*C1**2*(-3*C1**2 - 1) - 14*C1**2 + (-36*C1**2 - 5)*(-C1**2 - 1) + (-C1**2 - 1)*(-16*C1**2*(-3*C1**2 - 1) - 36*C1**2 + 8*(-9*C1**2 - 1)*(-C1**2 - 1) - 5) - 2)/120 + C1 + C1*x**2*(2*C1**2 + 1)/2 + C1*x**4*(-12*C1**2 + 8*(-3*C1**2 - 1)*(-C1**2 - 1) - 5)/24 + O(x**6))`, 

but the problem is that we want it to to return in terms of `erfi()` which will be easier to integrate in the next step to find the solution rather than simplifying it with its series expansion. 
[Wolfram's answer](https://www.wolframalpha.com/input/?i=+f%27%28x%29+%2B+f%28x%29**2+%2B+x*f%28x%29+%2B+1%3D0)

@oscarbenjamin, Kindly share your opinion.
Thanks.

Reply:
Ideally these series solutions would not be returned by `dsolve`